modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -348,14 +288,179 @@ processed.
 As said, you can wrap that command in a Cron job so it’s run every 5
 minutes for example.
 
+=== WND Listener Configuration
+
+Create a service following the instructions below that checks the share for changes:
+
+* For each WND mount point distinguished by a SERVER - SHARE pair,
+** place one copy of a file
+with following content under +
+`/etc/systemd/system/owncloud-wnd-listen-SERVER-SHARE.service`
+** replacing the all upper case words `SERVER`, `SHARE`, `USER` and `PASSWORD`
+** in both, the **filename** and in the **contents** below with their respective values. +
+Take care to also adjust the paths in `WorkingDirectory` and `ExecStart`
+according to your installation.
++
+....
+[Unit]
+Description=ownCloud WND Listener for SERVER SHARE
+After=syslog.target
+After=network.target
+Requires=apache2.service
+[Service]
+User=www-data
+Group=www-data
+WorkingDirectory=/var/www/owncloud
+ExecStart=/usr/bin/php ./occ wnd:listen -vvv SERVER SHARE USER PASSWORD
+Type=simple
+StandardOutput=journal
+StandardError=journal
+SyslogIdentifier=%n
+KillMode=process
+RestartSec=1
+Restart=always
+....
+
+* Run once for each created file the following commands:
++
+[source,console]
+----
+sudo systemctl enable owncloud-wnd-listen-SERVER-SHARE.service
+sudo systemctl start  owncloud-wnd-listen-SERVER-SHARE.service
+----
+
+=== WND Process Queue Configuration
+
+Create or add a `crontab` file in `/etc/cron.d/oc-wnd-process-queue`.
+
+* Make a `crontab` entry to run a script iterating over all `SERVER SHARE` pairs with
+an appropriate `occ wnd:process-queue` command. The commands must be **strictly sequential**.
```

Comment:
Just to clarify, if the process-queue overlaps with another execution of the same process-queue, this might be problematic due to possible race condition. However, you can run multiple process-queue as long as the host and share are different in all of them.
Basically, you should have one process-queue running per host and share.